build.sbt

```diff
@@ -1656,21 +1635,7 @@ lazy val `engine-runner` = project
           //          "-H:+DashboardAll",
           //          "-H:DashboardDump=runner.bgv"
           "-Dnic=nic"
-        ) ++ (if (
-                org.graalvm.polyglot.Engine
-                  .create()
-                  .getLanguages()
-                  .containsKey("java")
-              ) {
-                Seq(
-                  "-Dorg.graalvm.launcher.home=" + System.getProperty(
-                    "java.home"
-                  ),
-                  "--language:java"
```

Comment:
This probably removes a way to build [engine with Espresso](https://github.com/enso-org/enso/blob/826127d8ff36519b50a89c6dd3f607a8ea48bb32/docs/infrastructure/native-image.md#engine-with-espresso) - the documentation needs an update (at least) or the build script needs a fix.